pkg/manifests/config.go

```diff
@@ -66,6 +66,20 @@ func (c Config) IsStorageConfigured() bool {
 	return prometheusK8sConfig.VolumeClaimTemplate != nil
 }
 
+func (c Config) IsUserAlermanagerConfigMisconfigured() bool {
+	if c.ClusterMonitoringConfiguration != nil && c.UserWorkloadConfiguration != nil {
+		alertmanagerMainConfig := c.ClusterMonitoringConfiguration.AlertmanagerMainConfig
+		alertmanagerUWMConfig := c.UserWorkloadConfiguration.Alertmanager
+		if alertmanagerMainConfig != nil && alertmanagerUWMConfig != nil {
+			if alertmanagerMainConfig.EnableUserAlertManagerConfig && alertmanagerUWMConfig.Enabled &&
+				!alertmanagerUWMConfig.EnableAlertmanagerConfig {
```

Comment:
I would even consider that enabling AlertmanagerConfig resources in CMO configmap and user-defined Alertmanager in UWM configmap is enough to notify the user.

```suggestion
			if alertmanagerMainConfig.EnableUserAlertManagerConfig && alertmanagerUWMConfig.Enabled {
```

Reply:
Indeed! Especially because the default of `enableAlertmanagerConfig` is false!